test/components/LaunchDarkly.test.tsx

```diff
@@ -46,7 +46,7 @@ describe("components/LaunchDarkly", () => {
 
   it("renders the children", () => {
     const subject = shallow(
-      <LaunchDarkly clientId="080808" user="zeke">
+      <LaunchDarkly clientId="080808" user={{key: "zeke"}}>
```

Comment:
Has the interface for user been changed? Did it previously support both an object and a string or are these just poorly written tests to start with?

Reply:
This test file didn't have `@flow` included

Reply:
Looks like it's required as an object in https://docs.launchdarkly.com/sdk/client-side/javascript#section-users